assets/dev-env.nginx.template.conf.ejs

```diff
@@ -1,6 +1,29 @@
+<% if ( photon ) { %>
+
+location ^~ /wp-content/uploads/ {
+    expires max;
+    log_not_found off;
 <% if ( mediaRedirectDomain ) { %>
+    try_files $uri @prod_site;
+<% } %> 
+    location ~* \.(gif|jp?eg|png)$ {
+        fastcgi_pass photon:9000;
```

Comment:
Let's only pass the request to PHP in case there are any query arguments, otherwise it's a bit wasteful and will slow down the environment

Reply:
Done.